documentation/asciidoc/topics/ref_datagridservice_resources.adoc

```diff
@@ -11,9 +11,14 @@ spec:
   replicas: 6 <2>
   service:
     type: DataGrid <3>
-    container:
-      storage: 2Gi <4>
-    sites: <5>
+    container: <4>
+      extraJvmOpts: "-XX:NativeMemoryTracking=summary"
```

Comment:
@oraNod looks like extraJvmOpts, cpu and memory and storage parameters are from different container levels

Reply:
@dmvolod thanks, for some reason I thought `spec.container` and `spec.service.container` were dupes. I guess that's what happens when you have too many meetings in one day. fixed.

Reply:
No, but I'm also confusing `container's` sometimes. But not sure that's possible to change this structure dramatically...